pipelines/controller/src/main/java/com/google/fhir/analytics/DataProperties.java

```diff
@@ -105,6 +105,12 @@ public class DataProperties {
 
   private String fhirServerOAuthClientSecret;
 
+  private String fhirSinkPath;
```

Comment:
I think with adding this we need to add some more checks re. `dwhRoot`. The problem is that from the core pipeline perspective (which the controller calls), not setting `dwhRoot` is fine if `fhirSinkPath` is set. But this fails in the merger pipeline (since it depends on the Parquet files in the DWH). We can fix this by adding a constraint in `validateProperties` below (line 117) and enforce `dwhRoot` not to be null/empty.

However, this creates another more fundamental issue; let me start that with a question: In your scenario, do you care about Parquet files at all? If the answer is "no" then forcing Parquet file creation/merging (i.e., the current behavior in the controller) is a non-trivial overhead for the FHIR-to-FHIR scenario. For now, I suggest just adding the check in this PR, but if you want to avoid the overhead, we need to discuss more and that is another/larger change.

Reply:
Ill do that in a separate PR

Reply:
ill adres this in a separate PR

Reply:
Okay then please open an issue for it on GitHub.
Also I think it is more consistent to rename this config param to `sinkFhirServerUrl` for consistency.

Reply:
done https://github.com/google/fhir-data-pipes/issues/1073